README.md

```diff
@@ -204,58 +204,127 @@ USAGE:
     lychee [FLAGS] [OPTIONS] <inputs>...
 
 FLAGS:
-        --cache                  Use request cache stored on disk at `.lycheecache`
-        --dump                   Don't perform any link checking. Instead, dump all the links extracted from inputs that
-                                 would be checked
-    -E, --exclude-all-private    Exclude all private IPs from checking.
-                                 Equivalent to `--exclude-private --exclude-link-local --exclude-loopback`
-        --exclude-link-local     Exclude link-local IP address range from checking
-        --exclude-loopback       Exclude loopback IP address range and localhost from checking
-        --exclude-mail           Exclude all mail addresses from checking
-        --exclude-private        Exclude private IP address ranges from checking
-        --glob-ignore-case       Ignore case when expanding filesystem path glob inputs
-        --help                   Prints help information
-    -i, --insecure               Proceed for server connections considered insecure (invalid TLS)
-    -n, --no-progress            Do not show progress bar.
-                                 This is recommended for non-interactive shells (e.g. for continuous integration)
-        --offline                Only check local files and block network requests
-        --require-https          When HTTPS is available, treat HTTP links as errors
-        --skip-missing           Skip missing input files (default is to error if they don't exist)
-    -V, --version                Prints version information
-    -v, --verbose                Verbose program output
+        --cache                  
```

Comment:
Is the new `use_html5ever` flag messing with the help output here?

Reply:
afaict it changes indentation and padding everywhere, presumably due to its length

Reply:
It's because of the additional help text after the newline. We don't use that anywhere else. I think there was a way to tell structopt to render the short version without the additional lines and only render them when `--long-help` or something was used (?).

Reply:
TIL: 
> long_* variants are used when user calls the program with --help and “short” variants are used with -h flag.
https://docs.rs/structopt/latest/structopt/#help-messages

I wonder if we should use `-h` for the `README.md`; it gradually gets out of hand if not. 😆